alica_engine/src/engine/BasicBehaviour.cpp

```diff
@@ -33,6 +33,9 @@ BasicBehaviour::BasicBehaviour(const std::string& name)
     , success(false)
     , callInit(true)
     , started(true)
+    , _configuration(nullptr)
+    , msInterval(100)
```

Comment:
mismatch of variable naming convention: leading underscore

Reply:
Will be done when BasicBehavior is refactored...